nuopc.runseq

```diff
@@ -5,24 +5,24 @@ runSeq::
   MED med_phases_ocnalb_run
   MED med_phases_diag_ocn
   MED med_phases_prep_ice
-  MED -> ICE :remapMethod=redist
+  MED -> ICE :remapMethod=redist:srcTermProcessing=1:termOrder=srcseq
```

Comment:
I don't think these are necessary for reproducibility. Did you want to keep them in anyway?

Reply:
I've confirmed that `srcTermProcessing=1:termOrder=srcseq` aren't needed for reproducibility, so I've now removed them